### DISCUSSION TITLE
Would it be possible to add Proton-CachyOS to the mix?

### DISCUSSION THREAD

**Discussion by githlacker** [Vysp3r/ProtonPlus#272] [Ideas]
[Proton-CachyOS](https://github.com/CachyOS/proton-cachyos?tab=readme-ov-file) [(repo)](https://archive.cachyos.org/proton/) is a modified version of Proton developed by the creators of the [CachyOS](https://cachyos.org/) distribution. I've had a positive experience using it and would recommend considering its addition, if possible.

**Comment by Vysp3r** [comment]
For sure, but I am currently unavailable for quite some time.
Just make a feature request so I don't lose track of that.

**Reply by githlacker** [reply]
im going to past it there then thx for the reply
